docs/projects/lottie/layers/rectangle_layer.rst

```diff
@@ -0,0 +1,46 @@
+Rectangle Layer
+===============
+
+Table of Mappings
+-------------------
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       | layers/fill.json -> “o” -> properties/[value.json OR valueKeyframed.json] |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |            layers/shape.json -> “bm” -> helpers/blendMode.json            |
++---------------------+---------------------------------------------------------------------------+
+|        Color        |                      layers/fill.json -> “ty” = “fl”                      |
++---------------------+---------------------------------------------------------------------------+
+|       Point 1       |             Used in calculation of Origin parameter in lottie             |
```

Comment:
Can you add an image showing points 1 and 2? It will be nice, just to understand where they are. Is they top left or bottom right, or maybe in center?

Reply:
Done. :slightly_smiling_face: